routes/userRouter.js

```diff
@@ -5,7 +5,7 @@ const { loginRequired } = require("../utils/auth");
 const router = express.Router();
 
 router.post("/signin", userController.signIn);
-router.post("/update", loginRequired, userController.userUpdate);
+router.post("/update", loginRequired, userController.updateUser);
```

Comment:
RESTful에 맞게 동사는 사용하지 않습니다. HTTP method로 충분히 이해가 갈거에요
```suggestion
router.post("", loginRequired, userController.updateUser);
```